add note about licensing (#984)
MicrosoftDocs/OfficeDocs-O365ITPro: Office365-Admin/usage-analytics/enable-usage-analytics.md

![Select Get started free to get Power Bi](../media/7916b7ac-c1c8-40c0-b076-9a96ef5e0eb7.png)
  
You can also expand **Products** to buy a version of Power BI. 

> >[!NOTE]

Reviewer: ```suggestion
> [!NOTE]
```

> >You need a Power BI Pro license to install, customize, and distribute a template app. Please see more at [Prerequisites](https://docs.microsoft.com/en-us/power-bi/service-template-apps-install-distribute?source=docs#prerequisites)

Reviewer: ```suggestion
> You need a Power BI Pro license to install, customize, and distribute a template app. For more information, please see [Prerequisites](https://docs.microsoft.com/power-bi/service-template-apps-install-distribute?source=docs#prerequisites).
```

> >You need a Power BI Pro license to share your content, and the people you share it with do too, or the content needs to be in a workspace in a [Premium capacity](https://docs.microsoft.com/en-us/power-bi/service-premium-what-is). 

Reviewer: ```suggestion
> You need a Power BI Pro license to share your content, and the people you share it with do too, or the content needs to be in a workspace in a [Premium capacity](https://docs.microsoft.com/power-bi/service-premium-what-is). 
```

    
2. Select **Reports**, and then choose to **Display anonymous identifiers**. This setting gets applied both to the usage reports as well as to the template app.
  

> 3. Select **Save changes**.S

Reviewer: ```suggestion
3. Select **Save changes**.
```
